### DISCUSSION TITLE
gradle querydsl mongo 5

### DISCUSSION THREAD

**Discussion by s998899** [querydsl/querydsl#2955] [Q&A]
Hi anyone can provide working example with gradle of querydsl 5  with mango for me i tried everything qclasses not generating

Edit :  

for general i am able to generate now  by adding this but i am unable to generate for morphia even when i added mophia dependecy getting enity not found exception
	implementation "com.querydsl:querydsl-mongodb:${queryDslVersion}"
	implementation "com.querydsl:querydsl-apt:${queryDslVersion}:general"
	annotationProcessor"com.querydsl:querydsl-apt:${queryDslVersion}:general"

**Comment by jwgmeligmeyling** [comment]
Were you using Querydsl 4 with Gradle before or are you trying to do this with Querydsl 5 from scratch?

**Reply by s998899** [reply]
i am trying from scratch

**Reply by ankur198** [reply]
any update?
